package.json

```diff
@@ -9,17 +9,19 @@
   },
   "scripts": {
     "postinstall": "./scripts/postinstall.sh",
-    "test": "./node_modules/jest-cli/bin/jest.js",
-    "lint": "./node_modules/eslint/bin/eslint.js src/"
+    "coveralls": "./node_modules/mocha/bin/mocha --require blanket --require tests/bootstrap.js --reporter mocha-lcov-reporter --compilers .:tests/compiler.js src/**/__tests__/*-test.js src/components/**/__tests__/*-test.js | ./node_modules/coveralls/bin/coveralls.js",
+    "test": "./node_modules/mocha/bin/mocha --require tests/bootstrap.js --compilers .:tests/compiler.js src/**/__tests__/*-test.js src/components/**/__tests__/*-test.js",
+    "lint": "./node_modules/eslint/bin/eslint.js src/ && echo '🍻 All good!'"
```

Comment:
I've recently been pointed towards the documentation for npm-scripts which automatically adds `./node_modules/.bin/` to PATH before executing our custom commands: https://docs.npmjs.com/misc/scripts#path - with that in mind, could we drop all this prefixing?